shogun/transfer/multitask/MultitaskClusteredLogisticRegression.cpp

```diff
@@ -118,19 +111,11 @@ bool CMultitaskClusteredLogisticRegression::train_machine(CFeatures* data)
 	options.tasks_indices = ((CTaskGroup*)m_task_relation)->get_tasks_indices();
 	options.n_clusters = m_num_clusters;
 
-#ifndef HAVE_CXX11
-	malsar_result_t model = malsar_clustered(
-		features, y.vector, m_rho1, m_rho2, options);
-
-	m_tasks_w = model.w;
-	m_tasks_c = model.c;
-#else
 	SG_WARNING("Clustered LR is unstable with C++11\n")
```

Comment:
looking at the semantics of this, I don' think this is a good change ... ;) if malsar doesnt work with c++11, then we should either fix it or delete it or?